howso/update_cases.amlg

```diff
@@ -435,6 +440,12 @@
 					case_values (current_value 1)
 					closest_cases_map (assoc)
 					total_weight 0
+					case_weight
+						(if weight_feature_index
```

Comment:
this will statement fails when weight_feature_index is a valid value of 0

Reply:
ah you're so right. Will make the condition a bit more explicit